prime-router/metadata/hl7_mapping/common/datatype/xtn-telecom.yml

```diff
@@ -14,12 +14,12 @@ elements:
         temp: ORN
```

Comment:
That would be if we were setting field 2, but we are setting field 3 there.

Reply:
Just to be clear, I'm talking about `xtn-use-code` which has `hl7Spec: [ '%{hl7TelecomPath}-2' ]`. I can't comment on the exact line because it wasn't changed in the PR, but looking at the element above this comment.

Reply:
Got it, I can change that, but should we confer with Victor or something before just changing that?

Reply:
Yea looks like it would break some integration tests, so probably best to check with Victor, but I'm guessing it would be fine if we just update the integration tests accordingly.

Reply:
@victor-chaparro What are your thoughts on this?

Reply:
Yeah, making that change makes sense to me. I had added a TODO on top of that element saying there were issues when doing that, but I can't remember why and can't think of why it would cause any issues.

Reply:
Cool, I will change it then, thank you!